source/GameAction.cpp

```diff
@@ -177,6 +185,8 @@ void GameAction::LoadSingle(const DataNode &child, const string &missionName)
 		else
 			child.PrintTrace("Error: Skipping invalid outfit quantity:");
 	}
+	else if(key == "outfittag" && hasValue)
```

Comment:
I agree with all the use-cases given in the discussion, but I'm not on board yet with the keywords and concepts.

Not sure how to do this better, maybe by calling it meta-data instead of tags?   Outfits don't really get tagged with being "an engine", or being "human tech" or "republic tech", but a manufacturer or an intended use-case sounds a bit more like meta-data.

Reply:
so i should just rename it?

Reply:
Also, if we have functionality to remove outfits, then I expect such functionality to work on this "metadata", but also on attributes, so the syntax should be something like:
```
remove outfits
	ships
		<query to select the ships affected>
	outfits
		<query to select the affected outfits, including options to use attributes and this metadata>
```

Reply:
Why attributes? Can you give an example of why a content creator would want to remove something based on an attribute rather than by "metadata"? 

Bearing in mind that if an outfit/ship is missing valid metadata that a content creator needs, they can add that metadata in the same PR that requires the use of it.

Reply:
Sure; A plugin creator that creates a mission where all Jump-drive Outfits get removed for some mission where the player can fly in-system to a nearby planet where he/she can buy a new jump-drive (The government of holy planet X confiscated all jump-drive capable equipment; you were warned not to land on this holy planet with active jump-drives. The moon near planet Y seems to sell expensive jump-drives which you can purchase if you wish to leave the system later).
The plugin creator cannot update the original outfit, and we already have attributes that provide us the metadata to determine which outfits contain jump-drive functionality, so no need to add this metadata once more.

Reply:
will look over it tomorrow